domains/auth/Auth.API/API/Controllers/RefreshController.cs

```diff
@@ -0,0 +1,33 @@
+using API.Options;
+using API.Utilities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+
+namespace API.Controllers;
+
+[Authorize]
+[ApiController]
+public class RefreshController : ControllerBase
+{
+    [HttpGet()]
+    [Route("auth/refresh")]
+    public ActionResult RefreshAccessToken(
+        IHttpContextAccessor accessor,
+        IOptions<TokenOptions> tokenOptions,
+        IUserDescriptMapper descriptMapper,
+        ITokenIssuer tokenIssuer)
+    {
+        var oldDescriptor = descriptMapper.Map(User) ?? throw new NullReferenceException($"UserDescriptMapper failed: {User}");
+        var token = tokenIssuer.Issue(oldDescriptor);
```

Comment:
This endpoint cannot simply re-issue. This endpoint handles the issue with not regressing an issued token to one where terms are not accepted. Note this will likely require looking into beginning to create/define scope claims, i.e. the ones that we will need to publish to a shared library eventually.